src/fragments/lib/auth/js/advanced.mdx

```diff
@@ -279,243 +259,104 @@ const SignInWithGoogle = () => {
-      },
-      { 
-        // the user object, you can put whatever property you get from the Auth0
-        // for example:
-        name, // the user name
-        email, // Optional, the email address
-        phoneNumber, // Optional, the phone number
-      } 
-    );
-
-    console.log(cred);
+     customCredentialsProvider.loadFederatedLogin({
+        domain,
+        token: idToken,
+      });
+      const fetchSessionResult = await fetchAuthSession(); // will return the AWS credentials
+      console.log("fetchSessionResult: ", fetchSessionResult); 
   } catch(err) {
     console.log(err);
   }
 }
 ```
-
-Step 3. Get the current user and current Credentials:
-
-```js
-const user = await Auth.currentAuthenticatedUser();
-console.log(user);
-const creds = await Auth.currentCredentials();
-console.log(creds);
-// await Auth.currentSession() does not currently support federated identities. Please store the auth0 session info manually(for example, store tokens into the local storage).
-```
-
-Step 4. You can pass a refresh handler to the Auth module to refresh the id token from `Auth0`:
-
-```js
-async function refreshToken() {
-  // refresh the token here and get the new token info
-  // ......
-
-  const data = {
-    token, // the token from the provider
-    expires_at, // the timestamp when the token expires (in milliseconds)
-    identity_id, // optional, the identityId for the credentials
-  };
-
-  return data;
-}
-
-Auth.configure({
-  refreshHandlers: {
-    'your_auth0_domain': refreshToken
-  }
-})
-```
+</InlineFilter>
 
 ## Lambda Triggers
 
-The CLI allows you to configure [Lambda Triggers](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools-working-with-aws-lambda-triggers.html) for your AWS Cognito User Pool.  These enable you to add custom functionality to your registration and authentication flows. [Read more](/cli/function)
+The CLI allows you to configure [Lambda Triggers](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools-working-with-aws-lambda-triggers.html) for your Cognito User Pool. These enable you to add custom functionality to your registration and authentication flows. [Read more](/cli/function)
 
 ### Pre Authentication and Pre Sign-up Lambda triggers
 
-If you have a Pre Sign-up or Pre Authentication Lambda trigger enabled, you can pass `validationData` as one the properties for `signUp` or `signIn`. This metadata can be used to implement additional validations around authentication, such as restricting the types of user accounts that can be registered.
+If you have a Pre Authentication Lambda trigger enabled, you can pass `clientMetadata` as an option for `signIn`. This metadata can be used to implement additional validations around authentication.
 
 ```ts
-import { Auth } from 'aws-amplify';
+import { signIn } from 'aws-amplify/auth';
 
-async function signIn() {
+async function handleSignIn(username, password) {
```

Comment:
we should add the type definition here. If customer have `implicitAny` to true it will give them an error